spec/requests/shopify/export_variant_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe 'Export a Spree Variant to Shopify' do
+  include_context 'shopify_exporter_helpers'
+  include_context 'shopify_helpers'
+
+  let(:spree_product) { create(:product, name: 'Product Name') }
+  let(:spree_variant) { create(:variant, product: spree_product, sku: 'susan') }
+
+  before do
+    export_product!(spree_product)
+  end
+
+  after do
+    # This will auto-destroy the variants, due to the shopify associations.
```

Comment:
Should this comment be here or in the method definition (presumably in the shared context file)?